controllers/controllers/resource/template.go

```diff
@@ -161,6 +161,7 @@ func (r *VsphereTemplate) TemplateResources(ctx context.Context, eksaCluster *an
 	return generateTemplateResources(templateBuilder, clusterSpec, workloadTemplateNames, kubeadmconfigTemplateNames, cpOpt)
 }
 
+// TODO: This method is currently not used. Need to add logic in reconciler for TinkerbellDatacenterKind
```

Comment:
I suggest you throw your name against the to do: `todo(pokearu): ...`. It acts as a BOM so you can search for todo's you're responsible for in the code base super easy.

Reply:
Sure! But actually this isn’t related to the current changes, I noticed we hadn’t implemented so I just added a comment in case we forget. This would come in when we introduce update/upgrade.